Introducing the new OLS look - blogpost (#604)
open-life-science/open-life-science.github.io: _posts/2023-04-18-Introducing-the-new-OLS.md

---
layout: post
title: Introducing the new OLS look
authors:
- jilaga
image: /images/rebranding-Jilaga/2023-04-18-open-by-design.png
photos:
  name: photo by Jilaga
---

Exciting news! We wrapped up our rebranding excercise by **refreshing our logo** and **introducing a new name for our mentoring program**; **Open Seeds**! The new logo reaffirms our core values **Growth**, **Community** and **Nurture**, with a cohesive **colour palette**. We're excited to launch our refreshed brand identity and continue our commitment to advancing open research and scholarship.

> > This post is written by Jilaga Nneoma, OLS’s rebranding Outreachy intern, summarising the incredible logo redesign she did over the end of 2022 and first two months of 2023. Now that her internship’s finished, you can contact her via email nneomajilaga@gmail.com . If you’re interested in snagging her incredible talents for your own work, or browse her portfolio [here](https://linktr.ee/jilaga) 

Reviewer: ```suggestion
> This post is written by Jilaga Nneoma, OLS’s rebranding Outreachy intern, summarising the incredible logo redesign she did over the end of 2022 and first two months of 2023. Now that her internship’s finished, you can contact her via email [nneomajilaga@gmail.com](mailto:nneomajilaga@gmail.com). If you’re interested in snagging her incredible talents for your own work, or browse her portfolio [here](https://linktr.ee/jilaga) 
```

## Greetings!

> It has been quite an exciting journey for us here at Open Life Science (OLS) as we navigate the challenges of expanding our brand identity. As you may have noticed, we recently conducted a rebranding exercise to help us create a new brand identity that aligns with our core values and reflect our growth to include a diverse community of researchers and scientists from a variety of fields.

Reviewer: ```suggestion
It has been quite an exciting journey for us here at Open Life Science (OLS) as we navigate the **challenges of expanding our brand identity**. As you may have noticed, we recently conducted a rebranding exercise to help us create a **new brand identity** that **aligns with our core values** and **reflect our growth** to include a **diverse community** of researchers and scientists from a variety of fields.
```

> Initially, we set out to rebrand our name and logo to better represent our mission and core values. However, after careful consideration and feedback from our community, we have decided to keep the acronym OLS as our name. This decision was reached because of the strong association our community members have with the OLS name and what it stands for.

Reviewer: ```suggestion
Initially, we set out to rebrand our name and logo to better represent our mission and core values. However, after careful consideration and feedback from our community, we have decided to **keep the acronym OLS as our name**. This decision was reached because of the strong association our community members have with the OLS name and what it stands for.
```

> Nevertheless, we remain committed to improving our brand identity and expanding our reach. As part of this effort, we have refreshed our logo to give it a more modern look while still retaining the essence of our identity.

Reviewer: ```suggestion
Nevertheless, we remain committed to improving our brand identity and expanding our reach. As part of this effort, we have **refreshed our logo** to give it a more modern look while still retaining the essence of our identity.
```

## The New Logo

![The image shows the previous Open Life Science logo on the left and the new logo on the right.](/images/rebranding-Jilaga/2023-04-18-logo-refresh.png "Refresh of the Open Life Science logo")

> The new design features a simple yet dynamic logo mark that conveys the concept of community, growth and nurture.

Reviewer: ```suggestion
The new design features a simple yet dynamic logo mark that conveys the **concept of community, growth and nurture**.
```

## The Shape

![The image shows the design of the new Open Life Science logo composed of two growing leaves, symbolizing the collaborative nature of OLS and the community it supports.](/images/rebranding-Jilaga/2023-04-18-logomark.png "New Open Life Science logo")

Plant seeding is a powerful growth metaphor that symbolises new beginnings, potential, and growth. Just as a seed needs the right conditions to germinate and grow, individuals and ideas also require the right environment and support to thrive.

> The logo mark consists of two seeding leaves, which were carefully selected to represent growth and nurture. The two leaves are designed to symbolise the collaborative nature of OLS and the community it supports. 

Reviewer: ```suggestion
The logo mark consists of two seeding leaves, which were carefully selected to represent **growth** and **nurture**. The two leaves are designed to symbolise the **collaborative nature** of OLS and the community it supports. 
```

> The joined [petioles](https://en.wikipedia.org/wiki/Petiole_(botany))  of the leaves represent the strong bond that exists between OLS and its community, where knowledge and ideas are shared freely, and individuals are encouraged to grow and thrive. They also mimic the openness in OLS. 

Reviewer: ```suggestion
The joined [petioles](https://en.wikipedia.org/wiki/Petiole_(botany)) of the leaves represent the **strong bond that exists between OLS and its community**, where knowledge and ideas are shared freely, and individuals are encouraged to grow and thrive. They also mimic the openness in OLS. 
```

The organic shapes of the leaves evoke a sense of natural growth and development.

## The Colour

![The image shows the old color palette on the left and the new one on the right, which consists of three colors: the first is pigment green, the second is a soft, stimulating green called spring bud, and finally a shade of blue called phthalo blue.](/images/rebranding-Jilaga/2023-04-18-colourrefresh.png "New color palette")

These colors were chosen to represent growth, vibrancy, and innovation.

> **Pigment green** is a fresh and vibrant shade, representing the growth and renewal that OLS aims to promote in the research community. 

Reviewer: Can you also add the HEX code for the colors? That would be really useful

> These colors were chosen to represent growth, vibrancy, and innovation.
> 
> **Pigment green** is a fresh and vibrant shade, representing the growth and renewal that OLS aims to promote in the research community. 
> 
> **Spring bud**, a soft and uplifting shade of green, represents the nurturing environment that OLS provides for its mentees. 
> 
> **Phthalo blue** is a bold and dynamic shade that represents the cutting-edge research and innovation that OLS is known for.

Reviewer: ```suggestion
These colors were chosen to represent growth, vibrancy, and innovation.

- **Pigment green** is a fresh and vibrant shade, representing the **growth** and renewal that OLS aims to promote in the research community. 
- **Spring bud**, a soft and uplifting shade of green, represents the **nurturing environment** that OLS provides for its mentees. 
- **Phthalo blue** is a bold and dynamic shade that represents the **cutting-edge research and innovation** that OLS is known for.
```

> Finally, the soft gradient at the tip of the leaves blend together in a way that creates a sense of continuity and fluidity while adding visual interest and complexity.

Reviewer: ```suggestion
Finally, the soft gradient at the tip of the leaves blend together in a way that creates a sense of **continuity and fluidity** while adding visual interest and complexity.
```

Together, these colors form a cohesive and dynamic palette that represents the spirit of OLS and our commitment to advancing open and inclusive research. 

> The new brand colors will be incorporated across all of OLS's platforms, from our website to all our  social media channels to create a unified and recognizable brand identity.

Reviewer: ```suggestion
The new brand colors will be incorporated across all of OLS's platforms, from our website to all our social media channels to create a unified and recognizable brand identity.
```

## Open Seeds Mentoring and Training Program

![The image shows the logo of the Open Life Sciences Mentoring and Training Program called Open Seeds.](/images/rebranding-Jilaga/2023-04-18-open-seeds.png "Open Life Science Mentoring and Training Program Logo")

> **Open Seeds** is the new name for the Open Life Science mentoring program, reflecting their commitment to growing and nurturing talent within our community. It represents our focus on providing support and resources to help new ideas and projects flourish, while also honouring the spirit of openness and collaboration that is central to our mission. 

Reviewer: ```suggestion
**Open Seeds** is the new name for the OLS mentoring program, reflecting their commitment to growing and nurturing talent within our community. It represents our focus on **providing support and resources** to help new ideas and projects flourish, while also honouring the **spirit of openness and collaboration** that is central to our mission. 
```

With Open Seeds, OLS is planting the seeds of innovation and cultivating a community of leaders who will shape the future of research and discovery.

> The new Open Seeds logo builds on the refreshed OLS logo, featuring the two seedling leaves to mimic growth sprout. The logo communicates the values of the Open Seeds mentoring program, providing a fresh and modern identity that represents the program's mission to nurture and grow open scholarship initiatives.

Reviewer: Should we display the Open Seeds logo too?

Reviewer: Can we then add it also in the post?

> ## Conclusion

Reviewer: We should add a paragraph about our strategy to deploy the new branding

## Greetings!

It has been quite an exciting journey for us here at Open Life Science (OLS) as we navigate the **challenges of expanding our brand identity**. As you may have noticed, we recently conducted a rebranding exercise to help us create a **new brand identity** that **aligns with our core values** and **reflect our growth** to include a **diverse community** of researchers and scientists from a variety of fields.

Initially, we set out to rebrand our name and logo to better represent our mission and core values. However, after careful consideration and feedback from our community, we have decided to **keep the acronym OLS as our name**. This decision was reached because of the strong association our community members have with the OLS name and what it stands for.

Nevertheless, we remain committed to improving our brand identity and expanding our reach. As part of this effort, we have **refreshed our logo** to give it a more modern look while still retaining the essence of our identity.

## The New Logo

> ![The image shows the previous Open Life Science logo on the left and the new logo on the right.](/images/rebranding-Jilaga/2023-04-18-logo-refresh.png "Refresh of the Open Life Science logo")

Reviewer: ```suggestion
![The image shows the previous Open Life Science logo on the left and the new logo on the right.](/images/rebranding-Jilaga/2023-04-18-logo-refresh.png "Refresh of the Open Life Science logo"){: width="80%" }
```
